src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -162,8 +162,8 @@ private void insertAllDrivers(Car car) {
         }
         String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?)";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement statement =
-                        connection.prepareStatement(query)) {
+                 PreparedStatement statement =
```

Comment:
why are you changing it?

Reply:
A better practice would be to modify the lines you need, but not change code style of already implemented stuff